### PR TITLE
Update managing-holds.md

### PR DESCRIPTION
**Pull request by StefanieBier**

### CODE REVIEW

File: microsoft-365/compliance/managing-holds.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ To create a non-custodial hold for an eDiscovery (Premium) case:
 12. Review your settings, and then click **Create this hold**.
 
 > [!NOTE]
-> When you create a query-based hold, all content from selected locations is initially placed on hold. Subsequently, any content that doesn't match the specified query is cleared from the hold every seven to 14 days. However, a query-based hold won't clear content if more than five holds of any type are applied to a content location, or if any item has indexing issues.
+> When you create a query-based hold, all content from selected locations is initially placed on hold. Once the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. Once the character count across all queries on a single location exceed 10,000 characters, the entire location is placed on hold.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/managing-holds.md]:
```suggestion
> When you create a query-based hold, all content from selected locations is initially placed on hold. Once the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. Once the character count across all queries on a single location exceeds 10,000 characters, the entire location is placed on hold. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @StefanieBier, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@StefanieBier Gentle reminder, please implement the suggested changes, thanks.

**Comment by yogkumgit** [comment]
@tiburd Please help. Thanks

File: microsoft-365/compliance/managing-holds.md (hunk 1 above)
Comment (tiburd) [inline microsoft-365/compliance/managing-holds.md]:
```suggestion
> When you create a query-based hold, all content from selected locations is initially placed on hold. Once the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. After the character count across all queries on a single location exceeds 10,000 characters, the entire location is placed on hold. 
```

File: microsoft-365/compliance/managing-holds.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ To create a non-custodial hold for an eDiscovery (Premium) case:
 12. Review your settings, and then click **Create this hold**.
 
 > [!NOTE]
-> When you create a query-based hold, all content from selected locations is initially placed on hold. Subsequently, any content that doesn't match the specified query is cleared from the hold every seven to 14 days. However, a query-based hold won't clear content if more than five holds of any type are applied to a content location, or if any item has indexing issues.
+> When you create a query-based hold, all content from selected locations is initially placed on hold. Once the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. After the character count across all queries on a single location exceeds 10,000 characters, the entire location is placed on hold.
```
Comment (tiburd) [inline microsoft-365/compliance/managing-holds.md]:
```suggestion
> When you create a query-based hold, all content from selected locations is initially placed on hold. After the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. After the character count across all queries on a single location exceeds 10,000 characters, the entire location is placed on hold. 
```

**Pull request merged**